design/invitation.go

```diff
@@ -0,0 +1,59 @@
+package design
+
+import (
+	d "github.com/goadesign/goa/design"
+	a "github.com/goadesign/goa/design/apidsl"
+)
+
+var _ = a.Resource("invitation", func() {
+
+	a.BasePath("/invitations")
+
+	a.Action("createGroupInvite", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.POST("group/:inviteTo"),
+		)
+		a.Params(func() {
+			a.Param("inviteTo", d.String, "Unique identifier of the organization, team or security group")
+		})
+		a.Description("Create a new invitation for a user to join an organization, team or security group")
+		a.Payload(CreateInvitationRequestMedia)
+		a.Response(d.Unauthorized, JSONAPIErrors)
+		a.Response(d.Created)
+		a.Response(d.InternalServerError, JSONAPIErrors)
+		a.Response(d.BadRequest, JSONAPIErrors)
+	})
+
+	a.Action("list", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.GET(""),
+		)
+		a.Description("Lists invitations for an organization, team or group that the user has the invite_user scope for, or lists invitations for the current user")
+		a.Response(d.Unauthorized, JSONAPIErrors)
+		//a.Response(d.OK, invitationArray)
```

Comment:
Let's remove this from now if we are not implementing it in this PR. Or at least return NotImplemented() for now.

Reply:
controller returns NotImplemented() now